k8s/openebs-operator.yaml

```diff
@@ -152,6 +158,29 @@ spec:
         - name: MAYA_PORTAL_URL
           value: "https://mayaonline.io/"
 ---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: snapshot-controller
```

Comment:
Rename snapshot-controller to openebs-snapshot-controller. Please make the change in the below sections as well.

Reply:
Done !